GPU/GPUCommon.cpp

```diff
@@ -1075,6 +1081,22 @@ void GPUCommon::Execute_End(u32 op, u32 diff) {
 	}
 }
 
+void GPUCommon::Execute_TexScaleU(u32 op, u32 diff) {
+	gstate_c.uv.uScale = getFloat24(op);
+}
+
+void GPUCommon::Execute_TexScaleV(u32 op, u32 diff) {
+	gstate_c.uv.vScale = getFloat24(op);
+}
+
+void GPUCommon::Execute_TexOffsetU(u32 op, u32 diff) {
+	gstate_c.uv.uOff = getFloat24(op);
+}
+
+void GPUCommon::Execute_TexOffsetV(u32 op, u32 diff) {
+	gstate_c.uv.vOff = getFloat24(op);
```

Comment:
How can we avoid dirtying DIRTY_UVSCALEOFFSET here?  Just luck?

It seems like this would fail if a game drew a bezier, changed the U and V offset, then drew another bezier without changing texture parameters.

-[Unknown]